app.yaml

```diff
@@ -1,4 +1,4 @@
-runtime: nodejs12
+runtime: nodejs14
```

Comment:
The runtime doesn't matter, since it's just serving static files; however, we keep this in sync because we have to choose _something_, so it might as well be consistent.